docs/python/editing.md

```diff
@@ -56,10 +65,12 @@ For more on IntelliSense generally, see [IntelliSense](/docs/editor/intellisense
 
 ### Troubleshooting IntelliSense
 
-If autocomplete and IntelliSense aren't working for a custom module, check the following causes:
+For autocomplete and IntelliSense issues, check the following causes:
 
 | Cause | Solution |
 | --- | --- |
+| Pylance seems slow or consuming too much memory when working on a large workspace. | If there are subfolders you know can be excluded from Pylance's analysis, you can add their path to the `python.analysis.exclude` setting to see if performance improves. Alternatively, you can try setting  `python.analysis.indexing` to `false` to disable Pylance's indexer (**Note**: this will also impact the experience of completions and auto imports. Learn more about indexing in [code analysis settings](/docs/python/settings-reference.md#code-analysis-settings)).  |
```

Comment:
```suggestion
| Pylance seems slow or consuming is too much memory when working on a large workspace. | If there are subfolders you know can be excluded from Pylance's analysis, you can add their paths to the `python.analysis.exclude` setting to see if performance improves. Alternatively, you can try setting `python.analysis.indexing` to `false` to disable Pylance's indexer (**Note**: this will also impact the experience of completions and auto imports. Learn more about indexing in [code analysis settings](/docs/python/settings-reference.md#code-analysis-settings)).  |
```